transition to the PP 20170801

deniak opened the pull request:
Support for the new patent policy that was released last August.
The PR allows the use of the old patent policy until the end of this month. This should give enough time to update the spec generators.
Fix #696 

cc @plehegar, @marcoscaceres, @tabatkins

marcoscaceres commented:
Ok, sent: 
https://github.com/w3c/respec/pull/1388 

Tagged @deniak and @plehegar for review.

plehegar commented:
@wseltzer , what date would you like us to use for the text?

plehegar commented:
besides the date to check, LGTM

wseltzer commented:
@deniak and @plehegar it's still the same old patent policy, with some errata applied, so the date should be "5 February 2004 (updated 1 August 2017)"

wseltzer commented:
I think we might be getting too fancy here. Why not use the undated PP link https://www.w3.org/Consortium/Patent-Policy/ for everything except the primary "This document was produced by a group operating under the 5 February 2004 (Updated 1 August 2017) W3C Patent Policy."

plehegar commented:
if we're going to use the undated link version of the Patent Policy, can we also get rid of the date in the text and simply say:
This document was produced by a group operating under the <a href="https://www.w3.org/Consortium/Patent-Policy/">W3C Patent Policy</a>."
After all, we don't seem to make substantive changes to the policy (unlike the Process) so amybe we can simplify things further?

deniak commented:
> Shouldn't we edit `lib/rules.json` too, to update the [boilerplate suggested for rule  `sotd/pp, patPolReq`](https://www.w3.org/pubrules/doc/rules/?profile=WD#patPolReq)?

@tripu indeed. I'll update the rule after we agree on the link and text to use.

plehegar commented on test/docs/sotd/pp-20170801.html:
```diff
@@ -35,7 +35,7 @@
     <p>
       This document was produced by a group operating under the
-      <a href="https://www.w3.org/Consortium/Patent-Policy-20170801/">5 February 2004 (updated 1 August 2017)
+      <a href="https://www.w3.org/Consortium/Patent-Policy-20170801/">
```
should be https://www.w3.org/Consortium/Patent-Policy/

plehegar commented on lib/rules.json:
```diff
@@ -1324,7 +1324,7 @@
                     "changesList": "It <span class=\"rfc2119\">@{param1}</span> include a link to changes since the previous draft (e.g., a list of changes or a diff document or both; see the <a href=\"http://www.w3.org/2007/10/htmldiff\">online HTML diff tool</a>).@{param2}",
                     "stability": "It <span class=\"rfc2119\">must</span> set expectations about the (in)stability of the document. The <span class=\"rfc2119\">recommended</span> text @{param2}is: <blockquote class=\"boilerplate\"> <p>Publication as @{param1} does not imply endorsement by the W3C Membership. This is a draft document and may be updated, replaced or obsoleted by other documents at a ...
-                    "patPolReq": "It <span class=\"rfc2119\">must</span> include this text related to patent policy requirements (with suitable links inserted; see <a href=\"https://www.w3.org/2005/07/13-pubrules-disclosure\">guidelines for linking to disclosure pages</a>): <blockquote class=\"boilerplate\"> <p> This document was produced by a group operating under the <a href=\"https://www.w3.or ...
+                    "patPolReq": "It <span class=\"rfc2119\">must</span> include this text related to patent policy requirements (with suitable links inserted; see <a href=\"https://www.w3.org/2005/07/13-pubrules-disclosure\">guidelines for linking to disclosure pages</a>): <blockquote class=\"boilerplate\"> <p> This document was produced by a group operating under the <a href=\"https://www.w3.or ...
```
use https://www.w3.org/Consortium/Patent-Policy/

deniak commented:
@wseltzer, @plehegar, I commited the updated rule. Here's the markup we are looking for now:
```html
<p>
This document was produced by a group operating under the <a href="https://www.w3.org/Consortium/Patent-Policy/"> <abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy</a>. <abbr title="World Wide Web Consortium">W3C</abbr> maintains a <a href="https://www.w3.org/2004/01/pp-impl/40318/status" rel="disclosure">public list of
any patent disclosures</a> made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent. An individual who has actual knowledge of a patent which the individual believes contains <a href="https://www.w3.org/Consortium/Patent-Policy/#def-essential">Essential Claim(s)</a> must disclose the information in accordance with <a href="https://www.w3.org/Consortium/Patent-Policy/#sec-Disclosure">section 6 of the <abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy</a>.
</p>
```

Please confirm the markup is ok so we can deploy ASAP.

@tripu during a call today, we agreed to extend the transition period until end of Nov. I also addressed your comment about updating the rule description.
